### DISCUSSION TITLE
普林斯顿 COS126 计算机科学：跨学科方法 | hackway.org

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#25] [Announcements]
# 普林斯顿 COS126 计算机科学：跨学科方法 | hackway.org

课程名称： COS126 Computer Science: An Interdisciplinary Approach

https://hackway.org/docs/cs/intro

**Comment by zmgCoder** [comment]
感谢楼主无私奉献整理并公布。真是相见恨晚啊

**Reply by lidongyx** [reply]
谢谢支持，会持续不断更新完善
